doc/source/f2py/code/CMakeLists_skbuild.txt

```diff
@@ -16,74 +15,81 @@ if(PROJECT_SOURCE_DIR STREQUAL PROJECT_BINARY_DIR)
   )
 endif()
 
-# Grab Python
-find_package(Python3 3.9 REQUIRED
-  COMPONENTS Interpreter Development)
-
```

Comment:
Mixing FindPython and FindPythonInterp not a good idea! This will find the wrong Python.

Locally running this (without driving it via scikit-build) is a mess, but it's always going to be if you don't use FindPython. At least this now works.